plane-figures-circumference-calculator.js

```diff
@@ -0,0 +1,33 @@
+const circumference = (planeFigures) => {
+  if (planeFigures.shape === "circle") {
+    return `${2 * planeFigures.radius * Math.PI}`;
+  }
+  if (planeFigures.shape === "square") {
+    return `${planeFigures.side * 4}`;
+  }
+  if (planeFigures.shape === "rectangle") {
+    return `${2 * planeFigures.horizontalSide + 2 * planeFigures.verticalSide}`;
+  }
```

Comment:
You are converting numbers to string unnecessarily. You could just return the numbers directly:

```suggestion
  if (planeFigures.shape === "circle") {
    return 2 * planeFigures.radius * Math.PI;
  }
  if (planeFigures.shape === "square") {
    return planeFigures.side * 4;
  }
  if (planeFigures.shape === "rectangle") {
    return 2 * planeFigures.horizontalSide + 2 * planeFigures.verticalSide;
  }
```